src/components/Auth/ForgotPassword/ForgotPassword.react.js

```diff
@@ -136,6 +142,13 @@ class ForgotPassword extends Component {
           >
             {loading ? <CircularProgress size={24} /> : 'Reset'}
           </Button>
+          <LinkContainer>
+            <StyledLink
+              onClick={() => actions.openModal({ modalType: 'login' })}
+            >
+              <Translate text="Login Instead!" />
```

Comment:
Changes looks good! Please change the text to `Back to Login`.

Reply:
Oh.....sound good! 
and done  ✅